crates/snapbox/src/data.rs

```diff
@@ -145,6 +164,8 @@ impl Data {
                 Ok(data)
             }
             DataInner::Text(data) => Ok(data),
+            #[cfg(feature = "json")]
+            DataInner::Json(json) => Ok(serde_json::to_string(&json).unwrap()),
```

Comment:
`to_string` or `to_string_pretty`?

Reply:
`to_string_pretty` might be slightly better

Reply:
Especially for diffs